Wikipedia/Code/WMFNearbySectionController.h

```diff
@@ -5,6 +5,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+static NSString* const WMFNearbySectionIdentifier = @"WMFNearbySectionIdentifier";
```

Comment:
should be `extern` with the string defined in the implementation file (instead of a static exposed in the header)